### PR TITLE
add param to sidecar to ignore iptables changes

### PR DESCRIPTION
**Pull request by benraz**
This PR is intended to ignore iptables changes triggered by istio service (istio-start.sh)
The assumption that some users are commonly using iptables and would like to manage iptables manually instead of Istio.

### CODE REVIEW

**Comment by costinm** [comment]
@duderino  I think this is safe and useful in release-1.1 branch as well, since it fixes a clear problem for mesh expansion ( not working properly and messing up if iptable is already in use )

File: tools/deb/sidecar.env [hunk 1]
```diff
@@ -78,3 +78,7 @@
 
 # Location of istio configs.
 # ISTIO_CFG=/var/lib/istio
+
+# Ignore Istio iptables custom rules
+# Use this flag if you would like to manage iptables yourself Default to false (true/false)
```
Comment (incfly) [inline tools/deb/sidecar.env]: s/Use/Enable

s/yourself/yourself.

**Comment by benraz** [comment]
I signed it!

**Comment by benraz** [comment]
I signed it!

**Comment by john-a-joyce** [comment]
@benraz - this change caught my eye.   How should the Istio-cni behave in such an environment.  I guess if you are managing the Iptables yourself you would simply not install or enable the CNI.   Do you see any need to install the CNI but pay attention to this new parameter?   It is expected in the future for the CNI to become the default way to setup Iptables but I am sure an option will always exist to disable or not install it.

**Comment by tiswanso** [comment]
@benraz -- to add to @john-a-joyce 's Q's.  Wouldn't the same effect as this fix be just to install with the sidecar-injector configmap not having the istio-init container?  This would be identical to using the helm option `--set istio_cni.enabled=true`.  OR Is there some other advantage to using the istio-init container that you're trying to preserve?

**Comment by benraz** [comment]
hi @tiswanso The answer is yes, we are trying to avoid the installation and configuration via debian package, which is used for mesh-expansion feature, we are not manipulating the istio-sidecar within the containers, only trying to avoid the changes on our vms

**Pull request merged**